dockerfiles/sv-base/Dockerfile

```diff
@@ -53,8 +53,12 @@ RUN apt-get -qqy update --fix-missing && \
 #  4. final clean up
 COPY install_R_packages.R /opt/
 COPY install_bioconductor_packages.R /opt/
+COPY install_deprecated_R_package.sh /opt/
 ARG R_DEP_TRANSIENT="make gpg-agent gnupg"
-ARG R_DEPENDENCIES="software-properties-common build-essential ${R_DEP_TRANSIENT} libz-dev libncurses5-dev libbz2-dev liblzma-dev libcurl4-openssl-dev libssl-dev libxml2-dev ca-certificates apt-transport-https openssh-client"
+ARG R_DEPENDENCIES="software-properties-common build-essential \
+    ${R_DEP_TRANSIENT} libz-dev libncurses5-dev libbz2-dev liblzma-dev \
+    libcurl4-openssl-dev libssl-dev libxml2-dev ca-certificates \
+    apt-transport-https openssh-client curl"
```

Comment:
It looks like you added `curl` here, right? It looks like curl is already installed in `sv-base-mini`. However, it looks like it's getting listed as part of `GCLOUD_SDK_TRANSIENT_DEP` on line 16 of this dockerfile. I think that if you just removed it from that list, it would not get purged on line 36, and then would be available in this docker image (although I haven't tested that). That would avoid downloading and installing it twice.

Reply:
I did add curl.
I don't see the curl package in sv-base-mini (it looks like it install libcurl but not curl itself).
The other curl like you said is transient because it's needed temporarily for gcloud but not long-term. I was reluctant to make that a permanent dependency there because it might not be clear later on why it's permanent there. And curl is small. However it is a wasteful extra download, so if you think it's best I can make that change. Let me know what you think.

Reply:
I think curl is installed in the sv-base-mini dockerfile on line 61. It's available if you run an sv-base-mini image.

To my eye sv-base-mini is trying to install it permanently and then sv-base comes along and wipes it out unintentionally, which is why i suggested just removing it from the transient list here.

Reply:
You're right. My editor makes that block of the sv-base-mini dockerfile look like a bunch of comments, so I didn't check it carefully. I installed the dockerfile plugin and made the fix you suggested.